src/translations/en.json

```
@@ -5331,14 +5340,25 @@
               "confirm_text": "Are you sure you want to delete the entities?\n\nRemove them from your dashboard and automations if they include these entities.",
               "confirm_partly_text": "You can only delete {deletable} of the {selected} entities. The others require the integration to stop providing them, and sometimes a Home Assistant restart is needed. Are you sure you want to delete the deletable entities?\n\nRemove them from your dashboard and automations if they include these entities."
             },
+            "enable_recording_selected": {
+              "button": "Enable recording for selected",
+              "confirm_title": "Do you want to enable recording for {number} {number, plural,\n  one {entity}\n  other {entities}\n}?",
+              "confirm_text": "This will resume recording their state history in the recorder."
```

We can follow the wording in Core here as well by adding "database" to turn "Recorder" into its name:

```suggestion
              "confirm_text": "This will resume recording their state history in the Recorder database."
```